src/test/java/nextstep/subway/section/SectionAcceptanceTest.java

```diff
@@ -154,6 +153,124 @@ public void setUp() {
         assertThat(response.asString()).contains(Sections.HAS_NOT_UP_AND_DOWN_STATION_MSG);
     }
 
+    /**
+     * When 노선에 포함된 특정 역을 제거하면 (해당역이 노선의 중간일 때)
+     * Then 해당 역을 상행으로 포함한 구역이 제거되고 기존 구간이 변경된다. (HappyPath)
+     */
+    @Test
+    public void 특정역이_포함된_구간을_제거한다() {
+        //when
+        LineResponse 칠호선 = 노선을_생성한다("7호선", "#EEEEEE", 청담역.getId(), 건대역.getId(), 10).as(LineResponse.class);
+        구간을_생성한다(칠호선.getId(), 뚝섬유원지역.getId(), 건대역.getId(), 5);
+
+        //given
+        ExtractableResponse<Response> response = 구간을_제거한다(칠호선.getId(), 뚝섬유원지역.getId());
+
+        //then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
+
+        //when
+        ExtractableResponse<Response> response_line = 특정_노선목록을_조회한다(칠호선.getId());
```

Comment:
```suggestion
        ExtractableResponse<Response> responseLine = 특정_노선목록을_조회한다(칠호선.getId());
```

스네이크 케이스로 작성해주신 이유가 있을까요?